core-bundle/src/Doctrine/Backup/Config/AbstractConfig.php

```diff
@@ -43,32 +43,23 @@ public function isGzCompressionEnabled(): bool
         return $this->gzCompression;
     }
 
-    /**
-     * @return static
-     */
-    public function withGzCompression(bool $enable)
```

Comment:
How were you able to get this working with phpStan? I explicitly did this so it works.

Reply:
I have already reverted these changes. Guess we have to wait for PHP 8, so we can use `: static`.